packages/web/.env.production

```
@@ -3,6 +3,8 @@ ENV = 'production'
 
 # base api
 VUE_APP_BASE_API = './'
+VITE_APP_BASE_API = './'
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)